app/assets/sass/application.scss

```diff
@@ -24,3 +24,55 @@
     }
   }
 }
+
+.calendar-slots {
+  display: flex;
+  justify-content: space-between;
+  text-align: center;
+  width: 100%;
+
+  & > li {
+    flex-basis: 2rem;
+    height: 2rem;
+    line-height: 2rem;
+    padding: 0.5rem;
+  }
+
+  & a {
+    @include govuk-link-style-no-visited-state;
+    font-weight: bold;
+  }
+
+  & a:focus {
```

Comment:
Basically just turned off the yellow focus colouring here. It wasn't positioned correctly and was looking weird - figured it'd be fine for prototyping.